website/docs/d/data_factory.html.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_factory"
+sidebar_current: "docs-azurerm-datasource-data-factory-x"
+description: |-
+  Manages an Azure Data Factory (Version 2).
+---
+
+# Data Source: azurerm_data_factory
+
+Use this data source to access information about an existing Azure Data Factory (Version 2).
+
+## Example Usage
+
+```hcl
+
+data "azurerm_data_factory" "example" {
+  name            = "${azurerm_data_factory.example.name}"
+  resource_group_name = "${azurerm_data_factory.example.resource_group_name}"
+}
+
+output "data_factory_id" {
+value = "${azurerm_data_factory.example.id}"
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Data Factory. Changing this forces a new resource to be created. Must be globally unique. See the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/data-factory/naming-rules) for all restrictions.
```

Comment:
I think we shgould modify this for a data source
```suggestion
* `name` - (Required) Specifies the name of the Data Factory to retrieve information about.
```

Reply:
Hit undo too many times. Thanks for catching it for me!